packages/common/src/models/Analytics.ts

```diff
@@ -1675,6 +1691,28 @@ type BuyUSDCAddFundsManually = {
   eventName: Name.BUY_USDC_ADD_FUNDS_MANUALLY
 }
 
+// Buy Crypto
+
+type BuyCryptoEvent = {
+  eventName:
+    | Name.BUY_CRYPTO_STARTED
+    | Name.BUY_CRYPTO_ON_RAMP_OPENED
+    | Name.BUY_CRYPTO_ON_RAMP_SUCCESS
+    | Name.BUY_CRYPTO_ON_RAMP_FAILURE
+    | Name.BUY_CRYPTO_ON_RAMP_CANCELED
+    | Name.BUY_CRYPTO_ON_RAMP_CONFIRMED
+    | Name.BUY_CRYPTO_SUCCESS
+    | Name.BUY_CRYPTO_FAILURE
+    | Name.BUY_CRYPTO_RECOVERY_STARTED
+    | Name.BUY_CRYPTO_RECOVERY_FAILURE
+    | Name.BUY_CRYPTO_RECOVERY_SUCCESS
+
+  provider: string
+  amount: number
```

Comment:
nit: This is `requestedAmount` in the buy usdc analytics. Might be helpful for the events to use the same property names to avoid confusion. (I also think Amplitude has some special logic when properties are shared across events)

Reply:
good call